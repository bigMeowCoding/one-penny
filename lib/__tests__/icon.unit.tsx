import React from "react";
import * as renderer from "react-test-renderer";
import Icon from "../icon/icon";
describe("icon组件", () => {
  it("render correctly", () => {
    const json = renderer.create(<Icon name="wechat" />).toJSON();
    expect(json).toMatchSnapshot();
  });
});
