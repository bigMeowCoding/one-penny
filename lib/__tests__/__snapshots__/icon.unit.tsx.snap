// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`icon组件 render correctly 1`] = `
<svg
  className="zyj-ui-icon"
>
  <use
    xlinkHref="#wechat"
  />
</svg>
`;
